rfc/graphql-schema/additionalTypeDefs/byPage/orders.ts

```diff
@@ -0,0 +1,62 @@
+export default /* GraphQL */ `
+  type OrderHistory {
+    id: ID!
+    status: OrderStatus!
+  }
+
+  extend type Order {
+    orderHistory: [OrderHistory!]!
+    poNumber: String
+    company: Company
+  }
+
+  enum OrdersSortInput {
+    ID_ASC
+    ID_DESC
+    PO_NUMBER_ASC
+    PO_NUMBER_DESC
+    TOTAL_INC_TAX_ASC
+    TOTAL_INC_TAX_DESC
+    STATUS_ASC
+    STATUS_DESC
+    CREATED_AT_NEWEST
+    CREATED_AT_OLDEST
+  }
+
+  input CompanyOrdersFiltersInput {
+    dateRange: OrderDateRangeFilterInput
+    status: OrderStatusValue
+    # maybe replace this by a "customerId" field
+    email: String
+    createdBy: String
```

Comment:
what would this be? "Lee Dunkley"?

Reply:
yes

Reply:
again, feels odd as we have their IDs on hand

Reply:
it is, hence my comment of replacing it for `customerId` 😅